thisdot/lets-chat-with
# [Fix]: Implement Persistent New Connection Dismissal

# Overview

Resolves #19 

Dismisses the connection notification once either attendee dismisses it.

# Screenshots for Mobile and Desktop views (if applicable)

## Before

insert screenshot(s) here

## After

insert screenshot(s) here

# Details

## General

- List of changes
- Also list anything you haven't done or have deferred

## Automated Testing

- What unit tests have you added?
- What e2e/integration tests have you added?

### Manual Testing

Write the steps required for how might test this. Example:

_Note: requires two running instances of the app to get a match._

1. Perform Sign in and select the same conference
2. Connect with the first test users present (User 1 and 2 respectively in each instance)
3. Return to the connection view in both apps
4. See the notification for either or both
5. Dismiss the notification
6. Match with another user
7. See no notification
